rero_mef/agents/api.py

```diff
@@ -124,6 +126,13 @@ def reindex(self, forceindex=False):
             mef_record.reindex(forceindex=forceindex)
         return result
 
+    def add_information(self, resolve=False, sources=False):
```

Comment:
`resolve` and `sources` are unused, must be removed.
(or replace by `**kwargs` if this method could be override... and you want to keep same signature despite it's strange)

Reply:
Changed to:
```python
            rec = rec.add_information(
                resolve=request.args.get(
                    'resolve',
                    default=False,
                    type=lambda v: v.lower() in ['true', '1']
                ),
                sources=request.args.get(
                    'sources',
                    default=False,
                    type=lambda v: v.lower() in ['true', '1']
                )
            )
```